.github/workflows/e2e.yml

```diff
@@ -40,8 +40,8 @@ jobs:
 
       - name: Reclaim free space
         run: |
-          sudo swapoff -a
-          sudo rm -f /swapfile
+          # cleanup 30GB+ of tools we don't need for our CI
+          rm -rf /opt/{hostedtoolcache,ghc,az} /usr/share/dotnet /usr/share/swift
```

Comment:
did you try to remove dotnet with apt?

Reply:
As from the tmate session, removing all dotnet & mono packages seem to free only 4GB and not 20GB